styleguide/global/global.js

```diff
@@ -1,199 +1,172 @@
-      $el.find('.navbar-dropDownLink').toggleClass('navbar-dropDown-triggerActive');
-      //remove siblings open arrows
-      $el.siblings('.navbar-tertiary-dropDownTrigger').find('.navbar-dropDownLink').removeClass('navbar-dropDown-triggerActive');
-
-      $current.toggle();
-      $siblings.hide();
-    }
-  };
+  function dropDownTrigger(trigger) {
+    const $el = trigger;
+    const $current = $el.find('.navbar-tertiary-dropDownMenu');
+    const $siblings = $el.siblings('.navbar-tertiary-dropDownTrigger').find('.navbar-tertiary-dropDownMenu');
+    $el.find('.navbar-dropDownLink').toggleClass('navbar-dropDown-triggerActive');
+    // remove siblings open arrows
+    $el.siblings('.navbar-tertiary-dropDownTrigger').find('.navbar-dropDownLink').removeClass('navbar-dropDown-triggerActive');
+
+    $current.toggle();
+    $siblings.hide();
+  }
 
   // Indicate whether or not we're currently scrolling, or tapping on a link.
-  // Fix for known bootstrap scroll height bug: https://github.com/twbs/bootstrap/issues/12738
-  var fixScrollBug = function() {
-    $(".navbar-collapse").css({maxHeight: $(window).height() - $(".navbar-header").height() + "px"});
-  };
+  // Fix for known bootstrap scroll height bug: https:// github.com/twbs/bootstrap/issues/12738
+  function fixScrollBug() {
+    $('.navbar-collapse').css({ maxHeight: $(window).height() - `${$('.navbar-header').height()}px` });
+  }
+
   fixScrollBug();
-  window.addEventListener("resize", fixScrollBug);
+  window.addEventListener('resize', fixScrollBug);
 
-  //set the trigger to show drop downs on hover or clicks depending on viewport
-  var $viewPort;
-  $(".navbar-tertiary-dropDownTrigger")
-    .click(function(e){
+  // set the trigger to show drop downs on hover or clicks depending on viewport
+  let $viewPort;
+  $('.navbar-tertiary-dropDownTrigger')
+    .click((e) => {
       e.stopPropagation();
-      $viewPort = parseInt($(window).outerWidth());
-      if ($viewPort <= 768){
-        dropDownTrigger($(this));
+      $viewPort = parseInt($(window).outerWidth(), 10);
+      if ($viewPort <= 768) {
+        dropDownTrigger($(e.currentTarget));
       }
     })
-    .hover(function(){
-      $viewPort = parseInt($(window).outerWidth());
-      if ($viewPort > 768){
-        dropDownTrigger($(this));
+    .hover((e) => {
+      $viewPort = parseInt($(window).outerWidth(), 10);
+      if ($viewPort > 768) {
+        dropDownTrigger($(e.currentTarget));
       }
     });
 
-  //back to top
-  var offset = 250;
-  var duration = 300;
-  jQuery ('.back-to-top').click (function (event) {
-    event.preventDefault ();
-    jQuery ('html, body').animate ({scrollTop: 0}, duration);
-    return false;
-  });
-  jQuery (window).scroll (function () {
-    if (jQuery (this).scrollTop () > offset) {
-      jQuery ('.back-to-top').fadeIn (duration);
-    } else {
-      jQuery ('.back-to-top').fadeOut (duration);
-    }
-  });
```

Comment:
Yeah so the js has actually been broken on the scrolling effect for awhile and I wasnt sure why but there was a bug in to fix it. I think if we could fix it instead of removing it that would be super cool.